sources/packages/api/src/services/student/student.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable, Inject } from '@nestjs/common';
+import { RecordDataModelService } from '../../database/data.model.service';
+import { Student, User } from '../../database/entities';
+import { Connection } from 'typeorm';
+import { UserInfo } from '../../types';
+import { CreateStudentDto } from '../../route-controllers/student/models/student.dto';
+
+@Injectable()
+export class StudentService extends RecordDataModelService<Student> {
+  constructor(@Inject('Connection') connection: Connection) {
```

Comment:
Do we need to inject the connection here?
Could we inject the Repository<Student> instead?